applications/app/views/fragments/galleryBody.scala.html

```diff
@@ -8,42 +8,49 @@
 
 @(page: GalleryPage)(implicit request: RequestHeader, context: model.ApplicationContext)
 
-<div class="@RenderClasses(
-            Map("l-side-margins--paidgallery" -> isPaidContent(page)),
-            "l-side-margins", "l-side-margins--media", "l-side-margins--gallery"
-        )">
+<div class="gallery__main-content">
 
     <article id="article" class="@RenderClasses(
             Map("content--paidgallery" -> isPaidContent(page)),
-            "content", "content--media", "content--gallery", "tonal", "content--immersive", s"tonal--${toneClass(page.item)}"
+            "content", "content--media", "content--gallery", "tonal", "content--immersive", "tonal--tone-media", s"tonal--${toneClass(page.item)}"
```

Comment:
Doesn't tonal--tone-media already get added with `s"tonal--${toneClass(page.item)}`. If not, what does get added?